src/main/java/org/apache/maven/plugin/resources/remote/ProcessRemoteResourcesMojo.java

```diff
@@ -944,12 +944,12 @@ private void fileWriteIfDiffers( DeferredFileOutputStream outStream )
         }
     }
 
-    private MavenFileFilterRequest setupRequest( File source, File file, boolean isFiltering )
+    private MavenFileFilterRequest setupRequest( File source, File file )
     {
         MavenFileFilterRequest req = new MavenFileFilterRequest();
         req.setFrom( source );
         req.setTo( file );
-        req.setFiltering( isFiltering );
+        req.setFiltering( true );
```

Comment:
But this filtering ***always*** active, is that what we want?

Reply:
The property `useResourceFiltering` is still adhered to. It's evaluated on line 1279.

Reply:
@rfscholte do we need more review here?